testutil/runbook.go

```diff
@@ -0,0 +1,71 @@
+package testutil
+
+import (
+	"fmt"
+	"html/template"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+// BenchmarkSet returns a server and a path pattern for benchmarking.
+func BenchmarkSet(t testing.TB, bookCount, stepCount int, body string) (*httptest.Server, string) {
+	const httpRunbookTmpl = `desc: Test using HTTP
+runners:
+  req: {{ .Endpoint }}
```

Comment:
I want to enable the trace function and make it a bench target.

Reply:
Why do you want to enable `trace` ?

Reply:
It would be a good idea to set a policy for adding benchmarks.

* Make sure you have enough features to measure in one benchmark in advance.
Possibly detect unexpected performance degradation due to modifications
It is difficult to prepare benchmarks in advance. Hard to know which functions affected performance.
* Add a separate benchmark when you find a function for which you want to improve performance.
The more benchmarks you add, the longer it takes to run the CI
Which function's performance has changed? Easy to understand

My point was that I envisioned the former, and my understanding is that you were thinking the latter from the feel of our exchange on X.